setup.py

```diff
@@ -32,4 +70,11 @@
         "Programming Language :: Python :: 3.8",
         "Topic :: Scientific/Engineering",
     ],
+    ext_modules=extensions
+    # cythonize(
```

Comment:
we don't want to just have this as commented code.  but we do need to keep the notes somewhere about the python dev module and the annotate=True...as those are helpful 'gotchas' that we'll avoid having to look up as we do more Cython